docs/getting_started.rst

```diff
@@ -14,13 +14,23 @@ Install with Conda
 ------------------
 .. _install-with-conda:
 
-`conda <http://www.continuum.io/blog/conda>`_ is a python package manager built for scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries and binary dependencies, which are critical for most scientific workflows. If you're a ``conda`` user, you can install OpenPathSampling by adding the omnia channel. If you're not a conda user, you should look into it. ::
+We recommend using `conda` to install OpenPathSampling.  `conda
+<http://www.continuum.io/blog/conda>`_ is a python package manager built for
+scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries
+and binary dependencies, which are critical for most scientific workflows.
+If you're a ``conda`` user, you can install OpenPathSampling by adding the
+omnia channel. ::
 
-To install OpenPathSampling with conda, use the following commands ::
+To install the most recent release of OpenPathSampling with conda, use the
+following commands ::
 
   $ conda config --add channels http://conda.binstar.org/omnia
```

Comment:
You just want to use `omnia` instead of `http://conda.binstar.org/omnia` now.

In fact, you may just tell them that they want to do `conda install -c omnia openpathsampling`